packages/cli/src/oclif/commands/scaffold.js

```diff
@@ -1,181 +1,116 @@
 const _ = require('lodash');
 const path = require('path');
-const colors = require('colors/safe');
 
 const { flags } = require('@oclif/command');
 
 const BaseCommand = require('../ZapierBaseCommand');
 const { buildFlags } = require('../buildFlags');
+
 const {
-  ensureDir,
-  fileExistsSync,
-  readFile,
-  writeFile
-} = require('../../utils/files');
-const { camelCase, snakeCase } = require('../../utils/misc');
-const { splitFileFromPath } = require('../../utils/string');
-
-// what is the `resources: {}` app definition point?
-const typeMap = {
-  resource: 'resources',
-  trigger: 'triggers',
-  search: 'searches',
-  create: 'creates'
+  plural,
+  updateEntryFile,
+  writeTemplateFile,
+  createTemplateContext
+} = require('../../utils/scaffold');
+
+const getNewFileDirectory = (action, test = false) =>
+  `${test ? 'test/' : ''}${plural(action)}`;
+
+/**
+ * both the string to `require` and later, the filepath to write to
+ */
+const getFilename = (directory, noun) =>
```

Comment:
I was having trouble being satisfied about where these tiny util functions should live. They're here, but they're a little redundant and I don't love them